demos/nextjs-ai/lib/auth0-ai.ts

```diff
@@ -4,7 +4,7 @@ import { auth0 } from "./auth0";
 
 const auth0AI = new Auth0AI();
 
-export const withCalendarFreeBusyAccess = auth0AI.withFederatedConnection({
+export const withTokenForConnection = auth0AI.withTokenForConnection({
```

Comment:
does it make sense to keep the `withCalendarFreeBusyAccess`, so that it's easier to read from the other file?
I'm on board with the `auth0AI.withTokenForConnection`, but it's nice to see the "translation" of what the `.withTokenForConnection()` is used for in this specific scenario, right?  the instance of the fn.

Reply:
I thought `withCalendarFreeBusyAccess` was weird to read, since the method is only configuring getting access token and has nothing to do with calender except for specifying the scope to validate maybe. So maybe `WithTokenForGoogleConnection` is better name IMO. WDYT?